tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "lib": ["esnext"],
     "incremental": true,
     "strict": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "types": ["jest"]
```

Comment:
If we only care about the `jest` ambient types, then why not delete all the the other `@types/...` dependencies?

Reply:
The others are applied when the matching package is imported, so they're still useful (eg - `@types/react`). 
Honestly, we shouldn't even be shoving the jest types and should do explicit imports, but that's for another day...

Reply:
Ah! Right. None of the others are ambient.

Reply:
One of the things that @JacobMGEvans noted was that Vitest required explicit imports.

Reply:
I would be extremely happy to move to vitest once it's feature complete, it's looking pretty great so far.